OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -212,5 +214,22 @@ void INotifyHarvesterAction.Undocked()
 		{
 			isDocking = false;
 		}
+
+		void INotifyExternalCapture.OnCapturing(Actor self, Actor captor, Player oldOwner, Player newOwner)
```

Comment:
It would be a better fit to have `ExternalCapturable` grant a condition that modders can choose to use to disable this (or any other) trait rather than hardcoding the behaviour in here explicitly.

This is more in line with how how we handle damage states (using conditions) vs one-off notifications, which are mostly hardcoded.